openapi/openapi.yaml

```diff
@@ -697,6 +706,8 @@ definitions:
         $ref: '#/definitions/SalesChannel'
       supportType:
         $ref: '#/definitions/SupportType'
+      entityType:
+        $ref: '#/definitions/EntityType'
```

Comment:
why entitytype on profile? didn't we agreed to put it on agreement only?

Reply:
removed